app/views/layouts/countdown.html.erb

```diff
@@ -1,56 +0,0 @@
-<!DOCTYPE html>
-<html><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
```

Comment:
There are a few other partials and such below this that were the same - didn't seem to be referenced anywhere.